src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/jpa/entity/TierCalculationEntity.kt

```diff
@@ -32,4 +33,19 @@ data class TierCalculationEntity(
   @Type(JsonType::class)
   @Column(columnDefinition = "jsonb")
   val data: TierCalculationResultEntity,
-)
+) {
+  override fun equals(other: Any?): Boolean {
```

Comment:
how come you had to do this?

Reply:
There were warnings about this [can cause severe performance and memory consumption issues]